Pod/Classes/UITextField+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UITextField+Extensions.swift
+//  Pods
+//
+//  Created by Christopher Jones on 3/24/16.
+//
+//
+
+import UIKit
+
+internal extension UITextField {
+
+    internal static var emptyTextFieldCharacter: String {
+        return "\u{202F}"
```

Comment:
It's a very short empty text character. Methods like `textField: shouldChangeCharactersInRange:` or `deleteBackward` aren't called if the text field is empty (the latter is called - but only when entered from a hardware keyboard). Therefor, this character is inserted and as soon as the user hits backspace, the character gets deleted - text.characters.count becomes 0 -> we can go to the previous text field.
PaymentKit used a slightly different (even shorter) character, which caused weird behavior when used in a right-aligned text field: The cursor jumped to the left when starting editing.

tl;dr:
It's a barely visible character for detecting "backspace actions".